### DISCUSSION TITLE
Theia stuck on loading screen

### DISCUSSION THREAD

**Discussion by sboudreaux96** [eclipse-theia/theia#9472] [Q&A] [answered]
Hello - upon building Theia locally and starting it I have noticed that it is stuck "loading". I am not sure what is going on as I dont see any errors or problems with the workspace.
![theia-loading-screen](https://user-images.githubusercontent.com/83592423/118003461-9a0b0d00-b316-11eb-99cf-2e31750fe017.PNG)

**Comment by vince-fugnitto** [comment, marked as answer]
@sboudreaux96 can you verify if you have any errors in the browser dev console?
Are you building a Theia-based application, or building and running `master`?

**Reply by sboudreaux96** [reply]
@vince-fugnitto I do have a couple errors in the dev console - I am running this off of master 
![image](https://user-images.githubusercontent.com/83592423/118014972-c4ae9300-b321-11eb-9324-43507d3d336f.png)

**Reply by vince-fugnitto** [reply]
@sboudreaux96 do you mind sharing your `package.json`, it don't seem to have issues off master.

**Reply by sboudreaux96** [reply]
Github will not allow me to share the file itself so here is the contents: 
{
  "private": true,
  "name": "@theia/monorepo",
  "version": "0.0.0",
  "engines": {
    "yarn": "1.0.x || >=1.2.1",
    "node": ">=12.14.1 <13"
  },
  "resolutions": {
    "**/@types/node": "12",
    "**/node-abi": "^2.18.0"
  },
  "devDependencies": {
    "@types/chai-string": "^1.4.0",
    "@types/jsdom": "^11.0.4",
    "@types/node": "12",
    "@types/sinon": "^2.3.5",
    "@types/temp": "^0.8.29",
    "@types/uuid": "^7.0.3",
    "@typescript-eslint/eslint-plugin": "^4.8.1",
    "@typescript-eslint/eslint-plugin-tslint": "^4.8.1",
    "@typescript-eslint/parser": "^4.8.1",
    "chai-string": "^1.4.0",
    "colors": "^1.4.0",
    "concurrently": "^3.5.0",
    "electron-mocha": "^8.2.0",
    "eslint": "^7.14.0",
    "eslint-plugin-deprecation": "^1.1.0",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-no-null": "^1.0.2",
    "eslint-plugin-no-unsanitized": "^3.1.4",
    "eslint-plugin-react": "^7.21.5",
    "if-env": "^1.0.4",
    "ignore-styles": "^5.0.1",
    "jsdom": "^11.5.1",
    "lerna": "^2.2.0",
    "nyc": "^15.0.0",
    "rimraf": "^2.6.1",
    "sinon": "^3.3.0",
    "temp": "^0.8.3",
    "tslint": "^5.12.0",
    "typedoc": "^0.17.7",
    "typedoc-plugin-external-module-map": "^1.2.1",
    "typescript": "^3.9.2",
    "uuid": "^8.0.0"
  },
  "scripts": {
    "preinstall": "node-gyp install",
    "postinstall": "node scripts/post-install.js",
    "prepare": "yarn prepare:references && yarn --cwd packages/core prepare && yarn prepare:build && yarn prepare:hoisting && yarn download:plugins",
    "prepare:references": "node scripts/compile-references.js",
    "prepare:build": "yarn build && run lint && run build \"@theia/example-*\" --stream --parallel",
    "prepare:hoisting": "theia check:hoisted -s",
    "clean": "yarn lint:clean && node scripts/run-reverse-topo.js yarn clean",
    "build": "tsc -b configs/root-compilation.tsconfig.json",
    "watch": "tsc -b configs/root-compilation.tsconfig.json -w",
    "lint": "run lint",
    "lint:clean": "rimraf .eslintcache",
    "lint:oneshot": "node --max-old-space-size=4096 node_modules/eslint/bin/eslint.js --cache=true \"{dev-packages,packages,examples}/**/*.{ts,tsx}\"",
    "docs": "rimraf gh-pages/docs/next && typedoc --tsconfig configs/typedoc-tsconfig.json --options configs/typedoc.json",
    "test": "yarn test:references && yarn test:theia && yarn test:electron && yarn test:browser",
    "test:references": "node scripts/compile-references --dry-run",
    "test:theia": "run test \"@theia/!(example-)*\" --stream --concurrency=1",
    "test:browser": "yarn rebuild:browser && run test \"@theia/example-browser\"",
    "test:electron": "yarn rebuild:electron && run test \"@theia/example-electron\"",
    "rebuild:clean": "rimraf .browser_modules",
    "rebuild:browser": "theia rebuild:browser",
    "rebuild:electron": "theia rebuild:electron",
    "rebuild:electron:debug": "DEBUG=electron-rebuild && yarn rebuild:electron",
    "publish": "yarn && yarn test && yarn publish:latest",
    "publish:latest": "lerna publish --exact && yarn publish:check",
    "publish:next": "yarn next:publish && yarn next:publish --skip-npm && yarn publish:check",
    "next:publish": "lerna publish --exact --canary=next --npm-tag=next --yes",
    "publish:check": "node scripts/check-publish.js",
    "start:browser": "yarn rebuild:browser && yarn --cwd examples/browser start",
    "start:electron": "yarn rebuild:electron && yarn --cwd examples/electron start",
    "download:plugins": "theia download:plugins"
  },
  "workspaces": [
    "dev-packages/*",
    "packages/*",
    "examples/*"
  ],
  "theiaPluginsDir": "plugins",
  "theiaPlugins": {
    "vscode-builtin-bat": "https://open-vsx.org/api/vscode/bat/1.45.1/file/vscode.bat-1.45.1.vsix",
    "vscode-builtin-clojure": "https://open-vsx.org/api/vscode/clojure/1.45.1/file/vscode.clojure-1.45.1.vsix",
    "vscode-builtin-coffeescript": "https://open-vsx.org/api/vscode/coffeescript/1.45.1/file/vscode.coffeescript-1.45.1.vsix",
    "vscode-builtin-configuration-editing": "https://open-vsx.org/api/vscode/configuration-editing/1.45.1/file/vscode.configuration-editing-1.45.1.vsix",
    "vscode-builtin-cpp": "https://open-vsx.org/api/vscode/cpp/1.45.1/file/vscode.cpp-1.45.1.vsix",
    "vscode-builtin-csharp": "https://open-vsx.org/api/vscode/csharp/1.45.1/file/vscode.csharp-1.45.1.vsix",
    "vscode-builtin-css": "https://open-vsx.org/api/vscode/css/1.45.1/file/vscode.css-1.45.1.vsix",
    "vscode-builtin-css-language-features": "https://open-vsx.org/api/vscode/css-language-features/1.45.1/file/vscode.css-language-features-1.45.1.vsix",
    "vscode-builtin-debug-auto-launch": "https://open-vsx.org/api/vscode/debug-auto-launch/1.45.1/file/vscode.debug-auto-launch-1.45.1.vsix",
    "vscode-builtin-docker": "https://open-vsx.org/api/vscode/docker/1.45.1/file/vscode.docker-1.45.1.vsix",
    "vscode-builtin-emmet": "https://open-vsx.org/api/vscode/emmet/1.45.1/file/vscode.emmet-1.45.1.vsix",
    "vscode-builtin-fsharp": "https://open-vsx.org/api/vscode/fsharp/1.45.1/file/vscode.fsharp-1.45.1.vsix",
    "vscode-builtin-go": "https://open-vsx.org/api/vscode/go/1.45.1/file/vscode.go-1.45.1.vsix",
    "vscode-builtin-groovy": "https://open-vsx.org/api/vscode/groovy/1.45.1/file/vscode.groovy-1.45.1.vsix",
    "vscode-builtin-grunt": "https://open-vsx.org/api/vscode/grunt/1.45.1/file/vscode.grunt-1.45.1.vsix",
    "vscode-builtin-gulp": "https://open-vsx.org/api/vscode/gulp/1.45.1/file/vscode.gulp-1.45.1.vsix",
    "vscode-builtin-handlebars": "https://open-vsx.org/api/vscode/handlebars/1.45.1/file/vscode.handlebars-1.45.1.vsix",
    "vscode-builtin-hlsl": "https://open-vsx.org/api/vscode/hlsl/1.45.1/file/vscode.hlsl-1.45.1.vsix",
    "vscode-builtin-html": "https://open-vsx.org/api/vscode/html/1.48.0-next.4a1bcdafe9/file/vscode.html-1.48.0-next.4a1bcdafe9.vsix",
    "vscode-builtin-html-language-features": "https://open-vsx.org/api/vscode/html-language-features/1.48.0-next.4a1bcdafe9/file/vscode.html-language-features-1.48.0-next.4a1bcdafe9.vsix",
    "vscode-builtin-ini": "https://open-vsx.org/api/vscode/ini/1.45.1/file/vscode.ini-1.45.1.vsix",
    "vscode-builtin-jake": "https://open-vsx.org/api/vscode/jake/1.45.1/file/vscode.jake-1.45.1.vsix",
    "vscode-builtin-java": "https://open-vsx.org/api/vscode/java/1.45.1/file/vscode.java-1.45.1.vsix",
    "vscode-builtin-javascript": "https://open-vsx.org/api/vscode/javascript/1.45.1/file/vscode.javascript-1.45.1.vsix",
    "vscode-builtin-js-debug": "https://open-vsx.org/api/ms-vscode/js-debug/1.49.8/file/ms-vscode.js-debug-1.49.8.vsix",
    "vscode-builtin-json": "https://open-vsx.org/api/vscode/json/1.46.1/file/vscode.json-1.46.1.vsix",
    "vscode-builtin-json-language-features": "https://open-vsx.org/api/vscode/json-language-features/1.46.1/file/vscode.json-language-features-1.46.1.vsix",
    "vscode-builtin-less": "https://open-vsx.org/api/vscode/less/1.45.1/file/vscode.less-1.45.1.vsix",
    "vscode-builtin-log": "https://open-vsx.org/api/vscode/log/1.45.1/file/vscode.log-1.45.1.vsix",
    "vscode-builtin-lua": "https://open-vsx.org/api/vscode/lua/1.45.1/file/vscode.lua-1.45.1.vsix",
    "vscode-builtin-make": "https://open-vsx.org/api/vscode/make/1.45.1/file/vscode.make-1.45.1.vsix",
    "vscode-builtin-markdown": "https://open-vsx.org/api/vscode/markdown/1.45.1/file/vscode.markdown-1.45.1.vsix",
    "vscode-builtin-markdown-language-features": "https://open-vsx.org/api/vscode/markdown-language-features/1.39.2/file/vscode.markdown-language-features-1.39.2.vsix",
    "vscode-builtin-merge-conflict": "https://open-vsx.org/api/vscode/merge-conflict/1.45.1/file/vscode.merge-conflict-1.45.1.vsix",
    "vscode-builtin-npm": "https://open-vsx.org/api/vscode/npm/1.45.1/file/vscode.npm-1.45.1.vsix",
    "vscode-builtin-node-debug": "https://open-vsx.org/api/ms-vscode/node-debug/1.44.8/file/ms-vscode.node-debug-1.44.8.vsix",
    "vscode-builtin-node-debug2": "https://open-vsx.org/api/ms-vscode/node-debug2/1.42.1/file/ms-vscode.node-debug2-1.42.1.vsix",
    "vscode-builtin-objective-c": "https://open-vsx.org/api/vscode/objective-c/1.45.1/file/vscode.objective-c-1.45.1.vsix",
    "vscode-builtin-perl": "https://open-vsx.org/api/vscode/perl/1.45.1/file/vscode.perl-1.45.1.vsix",
    "vscode-builtin-powershell": "https://open-vsx.org/api/vscode/powershell/1.45.1/file/vscode.powershell-1.45.1.vsix",
    "vscode-builtin-pug": "https://open-vsx.org/api/vscode/pug/1.45.1/file/vscode.pug-1.45.1.vsix",
    "vscode-builtin-python": "https://open-vsx.org/api/vscode/python/1.45.1/file/vscode.python-1.45.1.vsix",
    "vscode-builtin-r": "https://open-vsx.org/api/vscode/r/1.45.1/file/vscode.r-1.45.1.vsix",
    "vscode-builtin-razor": "https://open-vsx.org/api/vscode/razor/1.45.1/file/vscode.razor-1.45.1.vsix",
    "vscode-builtin-ruby": "https://open-vsx.org/api/vscode/ruby/1.45.1/file/vscode.ruby-1.45.1.vsix",
    "vscode-builtin-rust": "https://open-vsx.org/api/vscode/rust/1.45.1/file/vscode.rust-1.45.1.vsix",
    "vscode-builtin-scss": "https://open-vsx.org/api/vscode/scss/1.45.1/file/vscode.scss-1.45.1.vsix",
    "vscode-builtin-shaderlab": "https://open-vsx.org/api/vscode/shaderlab/1.45.1/file/vscode.shaderlab-1.45.1.vsix",
    "vscode-builtin-shellscript": "https://open-vsx.org/api/vscode/shellscript/1.45.1/file/vscode.shellscript-1.45.1.vsix",
    "vscode-builtin-sql": "https://open-vsx.org/api/vscode/sql/1.45.1/file/vscode.sql-1.45.1.vsix",
    "vscode-builtin-swift": "https://open-vsx.org/api/vscode/swift/1.45.1/file/vscode.swift-1.45.1.vsix",
    "vscode-builtin-theme-abyss": "https://open-vsx.org/api/vscode/theme-abyss/1.45.1/file/vscode.theme-abyss-1.45.1.vsix",
    "vscode-builtin-theme-defaults": "https://open-vsx.org/api/vscode/theme-defaults/1.45.1/file/vscode.theme-defaults-1.45.1.vsix",
    "vscode-builtin-theme-kimbie-dark": "https://open-vsx.org/api/vscode/theme-kimbie-dark/1.45.1/file/vscode.theme-kimbie-dark-1.45.1.vsix",
    "vscode-builtin-theme-monokai": "https://open-vsx.org/api/vscode/theme-monokai/1.45.1/file/vscode.theme-monokai-1.45.1.vsix",
    "vscode-builtin-theme-monokai-dimmed": "https://open-vsx.org/api/vscode/theme-monokai-dimmed/1.45.1/file/vscode.theme-monokai-dimmed-1.45.1.vsix",
    "vscode-builtin-theme-quietlight": "https://open-vsx.org/api/vscode/theme-quietlight/1.45.1/file/vscode.theme-quietlight-1.45.1.vsix",
    "vscode-builtin-theme-red": "https://open-vsx.org/api/vscode/theme-red/1.45.1/file/vscode.theme-red-1.45.1.vsix",
    "vscode-builtin-theme-solarized-dark": "https://open-vsx.org/api/vscode/theme-solarized-dark/1.45.1/file/vscode.theme-solarized-dark-1.45.1.vsix",
    "vscode-builtin-theme-tomorrow-night-blue": "https://open-vsx.org/api/vscode/theme-tomorrow-night-blue/1.45.1/file/vscode.theme-tomorrow-night-blue-1.45.1.vsix",
    "vscode-builtin-typescript": "https://open-vsx.org/api/vscode/typescript/1.45.1/file/vscode.typescript-1.45.1.vsix",
    "vscode-builtin-typescript-language-features": "https://open-vsx.org/api/vscode/typescript-language-features/1.45.1/file/vscode.typescript-language-features-1.45.1.vsix",
    "vscode-builtin-vb": "https://open-vsx.org/api/vscode/vb/1.45.1/file/vscode.vb-1.45.1.vsix",
    "vscode-builtin-icon-theme-seti": "https://open-vsx.org/api/vscode/vscode-theme-seti/1.45.1/file/vscode.vscode-theme-seti-1.45.1.vsix",
    "vscode-builtin-xml": "https://open-vsx.org/api/vscode/xml/1.45.1/file/vscode.xml-1.45.1.vsix",
    "vscode-builtin-yaml": "https://open-vsx.org/api/vscode/yaml/1.45.1/file/vscode.yaml-1.45.1.vsix",
    "vscode-editorconfig": "https://open-vsx.org/api/EditorConfig/EditorConfig/0.14.4/file/EditorConfig.EditorConfig-0.14.4.vsix",
    "vscode-eslint": "https://open-vsx.org/api/dbaeumer/vscode-eslint/2.1.1/file/dbaeumer.vscode-eslint-2.1.1.vsix",
    "vscode-references-view": "https://open-vsx.org/api/ms-vscode/references-view/0.0.47/file/ms-vscode.references-view-0.0.47.vsix"
  },
  "dependencies": {
    "generator-theia-extension": "^0.1.23",
    "yo": "^4.1.0"
  }
}

**Reply by vince-fugnitto** [reply]
@sboudreaux96 I'm not sure as to why you're experiencing the issues when attempting to build `master`, is there anything in the docs you might be missing? In the meantime I will do a fresh clone, and attempt a build.

- https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites
- https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#quick-start

_Update_:

A fresh clone and build of `master` worked well:

![image](https://user-images.githubusercontent.com/40359487/118018417-bebab100-b325-11eb-8723-4574d36a556e.png)

**Reply by sboudreaux96** [reply]
@vince-fugnitto  I have reinstalled theia and now that seems to work! I had reinstalled and followed the instructions again and a fresh clone worked! Thanks!
